### PR TITLE
Check if p_child is not null in Container.fit_child_rect()

### LINKED ISSUE #24725: Calling `Container.fit_child_in_rect()` with null node leads to undefined behavior
Godot 3.1 b60939be88d192b63798aec6e9b031d570048b8b

`Container.fit_child_in_rect()` doesn't check if its `p_child` argument is null, which leads to undefined behavior, eventually crashes. I found this while doing something else, writing it down here to not forget.

### PR DESCRIPTION
**Pull request by timoschwarzer**
Fixes #24725

### CODE REVIEW

File: scene/gui/container.cpp [hunk 1]
```diff
@@ -95,6 +95,7 @@ void Container::_sort_children() {
 
 void Container::fit_child_in_rect(Control *p_child, const Rect2 &p_rect) {
 
+	ERR_FAIL_COND(p_child != null);
```
Comment (akien-mga) [inline scene/gui/container.cpp]: That's wrong, it's going to through an error when `p_child != null`, i.e. when the method was called correctly.

Comment (timoschwarzer) [reply scene/gui/container.cpp]: Whoops, too used to assertions. Fixed.

File: scene/gui/container.cpp [hunk 2]
```diff
@@ -95,6 +95,7 @@ void Container::_sort_children() {
 
 void Container::fit_child_in_rect(Control *p_child, const Rect2 &p_rect) {
 
+	ERR_FAIL_COND(p_child == null);
```
Comment (akien-mga) [inline scene/gui/container.cpp]: `null` in GDScript... :)

Comment (akien-mga) [reply scene/gui/container.cpp]: You can use `!p_child`, which would likely be the most idiomatic in Godot's code base.

Comment (timoschwarzer) [reply scene/gui/container.cpp]: Fixed

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**